### ISSUE TITLE
"Sorry, my bot code is having an issue" with LUIS and Skype 

### ISSUE THREAD

**Issue by liiver** [Microsoft/BotBuilder#1198]
My microsoft AppId: 3d4d6c46-e2c9-4f41-8ef4-45c46936e98b
My bot Id: eldur3
My Luis App Id: 61ed50f8-52d7-449a-ad78-50f804988d0e

Something went severly broken between my Skype Bot and LUIS system. On every message I send to the bot, It will just respond with "Sorry, my bot code is having an issue". WebChat has proper responses, and so does the emulator. 

Error from dev.botframework.com Skype Channel Issues view:  There was an error sending this message to your bot: InternalServerError

Sometimes I discovered from luislog.txt that multiple instances of Intents were et to triggered: true
[luislog.txt](https://github.com/Microsoft/BotBuilder/files/462669/luislog.txt)

Some logs attached from emulator (error_from_emulator.txt and ), might be helpful
[error_from_emulator.txt](https://github.com/Microsoft/BotBuilder/files/462640/error_from_emulator.txt)

**Comment by jameslew** [comment]
Are you using Application Insights to monitor your bot?  Provisioning app insights for your bot (default is free) and providing the instrumentation key to the Bot Framework is a great way to help in diagnosis as we send events to your app insights instance.

Do you mind if I try talking to your bot from the portal?

From the BotFramework side, for your botid I'm seeing very few messages sent from Skype through the Bot Framework (three in the last couple hours).  Does that match your expectations?

**Comment by msft-shahins** [comment]
Based on the exception in your logs, it means that you have multiple context.Wait(...) calls in your code, probably in your LuisDialog intent handlers. [Here](https://github.com/Microsoft/BotBuilder/blob/44d9667878759f9676c615365a818525ffe00ebd/CSharp/Tests/Microsoft.Bot.Builder.Tests/ExceptionTests.cs#L74) is an example test case causing the `MultipleResumeHandlerException` you are seeing.

**Comment by liiver** [comment]
Hmm, okay, it seems the LUIS thingie itself went abit crazy, so I needed to get a new one. Now it works half way, meaning.
When I have my bot in Conversation, it works and responds to my messages.
But It does not work in my 1:1 conversation.
Is it possible to check somehow, my ID should be 29:1osdU5cKOA-V-qkHLZy8klQ_qX_c8R-tVCJe5x8cewH8
AppId is 3d4d6c46-e2c9-4f41-8ef4-45c46936e98b, BotHandle eldur3

**Comment by jchannon** [comment]
I have a very similar issue. My bot works fine in Slack but doesn't work in Skype. It had been working and all of a sudden just stopped.

AppId `61b92b54-8dc4-42a3-965f-04950f72ded9`
LuisId `1ea0d2b8-0db6-4cd6-b5c6-1f1923927826`
Bot Handle `HTTPStatusBot`

**Comment by sitecorian** [comment]
Hey,

Having a similar issue too, same error in AI. The bot works fine in webchat and via the simulator running with ngrok and pointing to the same endpoint in the cloud.

Did you solve the issue?

Thanks!

**Comment by jchannon** [comment]
I issued /deleteprofile command in Skype and it worked

**Comment by sitecorian** [comment]
I found my problem using handy remote debugging of Web App via VS :)
First issue is that my input parsing that was trying to extract an http endpoint was failing because Skype wraps it in an anchor tag, while web chat was giving me plain text. This one is on me.

Second issue - _may be_ a bug in skype chat bot system. I was sending this code to indicate `typing`, which works perfectly in Slack and in the simulator, but it blows up with Skype. I added try/catch for now.

```csharp

var typingMessage = context.MakeMessage();
typingMessage.Type = ActivityTypes.Typing;
typingMessage.Text = "";
await context.PostAsync(typingMessage);

```

Biggest lesson - you gotta be careful when targeting multiple channels.
